rosetta-source/src/main/rosetta/product-template-type.rosetta

```diff
@@ -23,6 +23,30 @@ import cdm.product.collateral.*
 
 import cdm.mapping.config.*
 
+choice Asset:  <"An Asset is defined something that can be owned and transferred in the financial markets. As a choice data type, one and only one of the  attributes must be used. Cash: An Asset that consists solely of a monetary holding in a currency. Commodity: An Asset comprised of raw or refined materials or agricultural products, eg gold, oil or wheat. DigitalAsset An Asset that exists only in digital form, eg Bitcoin or Ethereum; excludes the digital representation of other Assets. Instrument: An asset that is issued by one party to one or more others; Instrument is also a choice data type.">
+    Cash
+    Commodity
+    DigitalAsset
+    Instrument
+
+choice Instrument:  <"A type of Asset that is issued by one party to one or more others. ListedDerivative: A securitized derivative on another asset that is created by an exchange. Loan: An Asset that represents a loan or borrow obligation. Security: An Asset that is issued by a party to be held by or transferred to others.">
+    ListedDerivative 
+    Loan
+    Security
```

Comment:
When `Security` eventually extends `InstrumentBase` instead of `ProductBase`, including a more comprehensive `taxonomy` attribute to `InstrumentBase` would allow to replace the `ProductTaxonomy` dependency.